frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.unit.spec.tsx

```diff
@@ -108,6 +110,20 @@ async function setup({ model, collections = [], usedBy = [] }: SetupOpts) {
   return { modelUpdateSpy };
 }
 
+function getDatabaseObject(database: Database) {
+  return {
+    ...database.getPlainObject(),
+    tables: database.tables.map(getTableObject),
+  };
+}
+
+function getTableObject(table: Table) {
+  return {
+    ...table.getPlainObject(),
+    schema: table.schema_name,
+  };
+}
+
```

Comment:
WDYT about keeping these utils under `server-mocks`? I personally wasted too much time figuring out the `schema` vs `schema_name` thing. It'd be neat not to copy-paste these functions here and there